learning_journal_db/models/mymodel.py

```diff
@@ -0,0 +1,19 @@
+from sqlalchemy import (
+    Column,
+    Index,
+    Integer,
+    UnicodeText,
+)
+
+from .meta import Base
+
+
+class MyModel(Base):
+    __tablename__ = 'models'
+    id = Column(Integer, primary_key=True)
+    title = Column(UnicodeText)
+    date = Column(UnicodeText)
+    body = Column(UnicodeText)
+
+
+Index('my_index', MyModel.id, unique=True, mysql_length=255)
```

Comment:
You don't need to have an index on the ID. The ID is the primary key, so the table will already be indexed by this column